app/controllers/api/physical_servers_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  class PhysicalServersController < BaseController
+    def show
+      if params[:c_id]
+        physical_server = PhysicalServer.find(params[:c_id])
+        response_payload = physical_server.as_json
+        response_payload['host_id'] = case physical_server.host
+                                      when nil then nil
+                                      else physical_server.host.id
+                                      end
```

Comment:
Oh, @chessbyte [had a comment](https://github.com/ManageIQ/manageiq/pull/14026/files/1b116a67b1311b13dcd62b179cbabe6986e8380a#r107178742) a while ago about this `case` statement.

The case statement could be:

```ruby
response_payload['host_id'] = physical_server.host.try(:id)
```